web-console/package.json

```diff
@@ -60,6 +62,7 @@
     "monaco-editor": "^0.49.0",
     "new-github-issue-url": "^1.0.0",
     "oidc-client-ts-lumeris": "^1.0.0",
+    "oslllo-svg-fixer": "^5.0.0",
```

Comment:
do our svgs have a problem? if so we should well designer about it

Reply:
Not a problem per se - but these SVGs contain "stroke"-s that make it inconvenient to use them unless I process them to reduce "stroke"-s to plain "path". The unrelated analogy would be processing "svg" file into more convenient "png".
I could commit processed SVGs, but I rather keep the "svg fixing" pipeline to prevent the same issue with new design delivery / other third party SVGs.
I commit original SVGs because there is nothing "wrong" with them per se, they just need to be adapted for the use in UI

Reply:
ok is this something the design team can fix on their end so you don't have to do this? I assume this is a problem if we use this svgs in other contexts too?

Reply:
can you put a picture here of how it looks like with and without the fix so they can see it

Reply:
This is not a problem if we use these SVGs in other contexts
But it is for many third-party SVGs we might want to use as icons

Will describe the problem for design team tomorrow